bip-update-process.md

```diff
@@ -0,0 +1,705 @@
+* Rationale -- The rationale fleshes out the specification by describing what motivated the design and why particular
+  design decisions were made. It should describe related work and alternate designs that were considered. The rationale
+  should record relevant objections or important concerns that were raised and addressed as this proposal was developed.
+* Backward compatibility -- A section describing any backwards incompatibilities, their severity, and instructions how
+  implementers/users should deal with these incompatibilities.
+* Reference implementation -- Where applicable, a reference implementation, test vectors, and documentation must be
+  finished before the BIP can be given the status "Complete". Any or all of these can be provided either in the BIP or
+  as auxiliary files (see [Auxiliary Files](#auxiliary-files)).
+* Changelog -- A section to track modifications to a BIP after reaching Complete status.
+* Copyright -- The BIP must be placed under an acceptable license (see [BIP Licensing](#bip-licensing) below).
+
+#### BIP header preamble
+
+Each BIP must begin with an [RFC 822-style header preamble](https://www.w3.org/Protocols/rfc822/). The headers must
+appear in the following order. Headers marked with "\*" are optional. All other headers are required. The overview is
+followed by an explanation for each header.
+
+```
+  BIP: <BIP number, or "?" before assignment>
+* Layer: <Consensus (soft fork) | Consensus (hard fork) | Peer Services | API/RPC | Applications>
+  Title: <BIP title (≤ 50 characters)>
+  Authors: <Authors’ names and email addresses>
+* Shepherds: <Shepherds’ names and email addresses>
+  Status: <Draft | Complete | Deployed | Closed>
+  Type: <Specification | Informational | Process>
+  Created: <Date of number assignment (yyyy-mm-dd)>
+  License: <Identifier(s) of acceptable license(s)>
+* License-Code: <Identifier(s) for Code under different acceptable license(s)>
+* Discussion: <Mailing list thread(s), or other noteworthy discussion(s) in "date: URL" format>
+* Revision: <Version number (MAJOR.MINOR.PATCH)>
+* Requires: <BIP number(s)>
+* Replaces: <BIP number>
+* Superseded-By: <BIP number>
+```
+
+__BIP__: The BIP number, or "?" before a number was assigned. A proposal must not self-assign a BIP number. Numbers are
+assigned solely by BIP Editors.
+
+__Layer__: The layer of bitcoin the BIP applies to. See [BIP 123](bip-0123.mediawiki) for definitions of the various BIP
+layers.
+
+__Title__: A descriptive title of up to 50 characters.
+
+__Authors__: The authors header lists the names (or pseudonyms) and email addresses of all authors of the BIP. The
+format of each authors header value must be
+
+    Random J. User <address@dom.ain>
+
+If there are multiple authors, each should be on a separate line following [RFC
+2822](https://datatracker.ietf.org/doc/html/rfc2822.html) continuation line conventions.
```

Comment:
AFAICT RFC 2822 doesn't specify "continuation line conventions" with that phrasing. The word "continuation" only appears once in a footnote. Perhaps it should refer to the [Long Header Fields](https://www.rfc-editor.org/rfc/rfc2822#section-2.2.3) section? Otherwise, it's not entirely clear what the convention actually is, and whether that section describes the convention.

This sentence appears to be lifted from [PEP 1](https://peps.python.org/pep-0001/), which provides equally vague guidance.

Reply:
Okay good, I replaced it with 

```diff
-If there are multiple authors, each should be on a separate line following [RFC
-2822](https://datatracker.ietf.org/doc/html/rfc2822.html) continuation line conventions.
+Multiple authors should be recorded on separate lines:
+
+    Authors: Random J. User <address@dom.ain>
+             Anata Sample <anata@domain.example>
```